features/support/simplecov.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 require 'simplecov'
-SimpleCov.start 'rails'
+SimpleCov.start 'rails' do
+  add_filter "/channels/"
+  add_filter "/jobs/"
+  add_filter "/mailers/"
+  add_filter "admins_controller"
```

Comment:
I've added the testcases for this. Infact, just added this file changes as well to run my testcases lol.